eagleproject/core/blockchain/harvest/transaction_history.py

```diff
@@ -529,6 +575,26 @@ def fetch_supply_data(block_number):
         'other_non_rebasing': other_non_rebasing,
     }
 
+def fetch_ogv_data(to_block, from_timestamp, to_timestamp):
+    ogv_history = get_coin_history('OGV', from_timestamp, to_timestamp)
+    index = len(ogv_history['prices']) - 1
```

Comment:
nit: what happens if `len(ogv_history)` is 0? I'm wondering if this whole block of code should be wrapped in a try catch?

Reply:
added a try catch block